### DISCUSSION TITLE
[resolved] billing Zwrot kosztów powiązać z zamówieniem

### DISCUSSION THREAD

**Discussion by drzemik** [allegro/allegro-api#4733] [Suggestions - resolved]
:bulb: **Sugestia**
Proszę dla zasobu billing/billing-entries 
dla operacji billingowej REF zwrot kosztów przypisać numer zamówienia.

**Comment by PrzemyslawLukanowski** [comment]
Zgodnie z naszym [newsem](https://github.com/allegro/allegro-api/issues/4337), przenoszę Twoją sugestię do zakładki Discussions.

**Reply by drzemik** [reply]
Czy coś się dzieje z tym tematem?

**Reply by AureliuszBrussy** [reply]
Aktualnie nie pracujemy nad tą propozycją. Jeżeli zdecydujemy się na wdrożenie Twojej sugestii - poinformujemy o tym w tym wątku.

**Comment by MartaNowaczyk** [comment]
Zmieniam status na Resolved - zmiana wprowadzona - [news](https://developer.allegro.pl/news/get-billing-billing-entries-we-have-introduced-order-id-in-a-refund-of-costs-YW19dW1VKf1).
